### PR TITLE
Feedback Button in every page

### PR DESCRIPTION
**Pull request by anurag629**
Because

fix: Created and added Feedback button to every page.

### CODE REVIEW

File: src/components/docs/Nav/Nav.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const DropdownMenu = ({
     <Dropdown.Menu key={item.key} open={isOpen}>
       <Dropdown.MenuTrigger className="flex flex-row gap-x-1 focus:outline-none">
         <p
-          onClick={() => setIsOpen(!isOpen)}
+          onMouseEnter={() => setIsOpen(!isOpen)}
```
Comment (EiffelFly) [inline src/components/docs/Nav/Nav.tsx:54]: @anurag629 I discovered you had filed this change with different PR, could you move this commit to that PR? In this way we can better track the issue

https://github.com/instill-ai/instill.tech/pull/623

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -94,6 +95,7 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
       <InstillAICtxProvider>
         {getLayout(<Component {...pageProps} />)}
         <div id="zoomable-image" />
+        <Feedback />
       </InstillAICtxProvider>
```
Comment (EiffelFly) [inline src/pages/_app.tsx:99]: @anurag629 I am sorry that we didn't address this clearer. 

https://github.com/instill-ai/community/issues/394

In the issue, the main goal is putting this link on the documentation website along with other buttons on the leftSidebar. 

Could you take a look at this [component](https://github.com/instill-ai/instill.tech/blob/8cd1e761f1c6d90b1cd42305f53f990aef5805ea/src/components/ui/ContributeLinks.tsx#L11)

CleanShot 2023-10-06 at 15 12 40
 and use the style but change the link and wording. and put it under "Edit this page" link

**Review by EiffelFly — COMMENTED** [review]
@anurag629

Thanks for contributing to our repo, we are really appreciated it, please check my comment below.

**Comment by anurag629** [comment]
> But if there is no following update, we will close this PR. As

@EiffelFly, I was busy last week, I'll commit changes by tomorrow.

**Comment by EiffelFly** [comment]
Closed due to lack of activity

**Pull request closed without merging**